.nycrc.json

```diff
@@ -1,6 +1,7 @@
 {
   "exclude": [
     "support-utils.js",
-    "task-utils.js"
+    "task-utils.js",
+    "support.js"
```

Comment:
Why is this being excluded?

Reply:
If you run the unit-tests, support.js is reported as having too little coverage, but support.js is not even being tested (only support-utils.js). I thought that this is an issue with my code, but I get the same result on the main branch.